examples/k8s/value_tso_HA.yaml

```diff
@@ -0,0 +1,449 @@
+# Default values for byconity.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+# please replace  storageClassName
+nameOverride: ""
+fullnameOverride: ""
+
+byconity:
+  image: byconity/byconity:0.1.0-GA
+  imagePullPolicy: IfNotPresent
+  hdfs_address: hdfs://byconity-hdfs-namenodes:8020 # can using your own hdfs
+  #  use_existing_fdb: true
+  #fdb_cluster_file: byconity_fdb:Is0hBgl6iICdHuspBmhAODmD5WISXKzI@192.168.224.150:4501,192.168.226.83:4501,192.168.228.152:4501
+  ports:
+    tcp: 9000
+    http: 8123
+    rpc: 8124
+    tcpSecure: 9100
```

Comment:
we do not need tcp, tcpSecure, https, exchange, exchangeStatus port for TSO

Reply:
this is not for tso only. it's for all components